update-files.sh

```diff
@@ -43,3 +43,7 @@ curl -sf -o src/bci_build/package/postfix/entrypoint/ldap/virtual_mailbox_maps h
 curl -sf -o src/bci_build/package/postfix/entrypoint/ldap/virtual_uid_maps https://raw.githubusercontent.com/thkukuk/containers-mailserver/master/postfix/ldap/virtual_uid_maps
 
 patch src/bci_build/package/postfix/entrypoint/entrypoint.sh src/bci_build/package/postfix/entrypoint/sles-entrypoint.patch -o src/bci_build/package/postfix/entrypoint/entrypoint.sles.sh
+
+### podman: Apache-2
+curl -sf -o src/bci_build/package/podman/containers.conf https://raw.githubusercontent.com/containers/image_build/main/podman/containers.conf
+curl -sf -o src/bci_build/package/podman/podman-containers.conf https://raw.githubusercontent.com/containers/image_build/main/podman/podman-containers.conf
```

Comment:
please also fetch the license from the base of the repo and install it as EXTRA_FILE